commands/_morningAnnouncement.py

```diff
@@ -4,16 +4,17 @@
 
 
 async def send_morning_announcement(client, guild_id, channel_id):
-    role = discord.utils.get(client.get_guild(guild_id).roles, name = _mongoFunctions.get_announcement_role_string(guild_id))
+    print("this got called")
+    role = discord.utils.get(client.get_guild(guild_id).roles, name = _mongoFunctions.get_settings(guild_id)['announcement_role'])
     await client.get_guild(guild_id).get_channel(channel_id).send(role.mention, embed = _embedMessage.create("Good Morning!",
                                                                                                              _mongoFunctions.random_quote(guild_id,
-                                                                                                                                          _mongoFunctions.get_announcement_quoted_person(
-                                                                                                                                              guild_id)),
+                                                                                                                                          _mongoFunctions.get_settings(guild_id)[
+                                                                                                                                              'announcement_quoted_person']),
                                                                                                              "blue"))
     _mongoFunctions.set_last_announcement_time(guild_id, datetime.now())
 
```

Comment:
If we're revamping settings, can we make it an option to disable morning announcements? I doubt it will get used on any servers except for ours tbh, might wanna keep it as something in schedulejobs()

Reply:
Should be a quick fix ill make an issue for it